pages/common/neo4j-admin.md

```diff
@@ -0,0 +1,32 @@
+# neo4j-admin
+
+> Manage and administer a Neo4j DBMS.
+> More information: <https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/>.
```

Comment:
```suggestion
> See also: `mysql`, `sqllite3`, `psql`, `usql`.
> More information: <https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/>.
```
Non-blocking (doesn't prevent merging) change: we support page suggestions, it would be nice to recommend similar tools, who looks up for DBMS commands can be interested in other similar tools.

Reply:
Im hoping to add additional neo4j commands, so I'll add a suitable "See also:" at that time if that suits.

Reply:
Awesome! They will be a better "See also" indeed, we can leave just one different DBMS for connecting with other DBMSes.